src/Render/Renderer.php

```diff
@@ -0,0 +1,365 @@
+<?php
+
+namespace SilverStripe\DeprecationChecker\Render;
+
+use Doctum\Project;
+use Doctum\Version\Version;
+use InvalidArgumentException;
+use LogicException;
+use SilverStripe\DeprecationChecker\Compare\BreakingChangesComparer;
+use Symfony\Component\Filesystem\Path;
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+/**
+ * Renderer that takes API breaking changes and renders a twig template.
+ */
+class Renderer
+{
+    private array $metaDataFrom;
+
+    private array $metaDataTo;
+
+    private Project $parsedProject;
+
+    public function __construct(array $metaDataFrom, array $metaDataTo, Project $parsedProject)
+    {
+        $this->metaDataFrom = $metaDataFrom;
+        $this->metaDataTo = $metaDataTo;
+        $this->parsedProject = $parsedProject;
+    }
+
+    /**
+     * Render the changelog chunk for these API breaking changes
+     */
+    public function render(array $breakingApiChanges, string $baseDir, string $filePath): void
+    {
+        $this->parsedProject->switchVersion(new Version(BreakingChangesComparer::TO));
+        $data = [
+            'fromVersion' => $this->metaDataFrom['branch'],
+            'toVersion' => $this->metaDataTo['branch'],
+            'apiChanges' => $this->getFormattedApiChanges($breakingApiChanges),
+        ];
+        $loader = new FilesystemLoader(Path::join(__DIR__, '../../templates'));
+        $cacheDir = Path::join($baseDir, 'cache/twig');
```

Comment:
Put `cache/twig` into a class const so that it's a little clearer

Reply:
Done